src/index.ts

```diff
@@ -50,9 +64,7 @@ Array.prototype.max = function (func = x => x, thisArg = this) {
 Array.prototype.shuffle = function (thisArg = this) {
   return thisArg
     .map(val => ({
-      sort:
-        Math.floor(Math.pow(10, 14) * Math.random() * Math.random()) %
-        (thisArg.length * 100),
+      sort: Math.floor(1e14 * Math.random() ** 2) % (thisArg.length * 100),
```

Comment:
Why this exact formula? It will give a random value, though, so all's fine. Just curious.